lib/Dashboard/TalkPanel.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * @copyright Copyright (c) 2020 Julius Härtl <jus@bitgrid.net>
+ *
+ * @author Julius Härtl <jus@bitgrid.net>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Talk\Dashboard;
+
+use OCP\IL10N;
+
+class TalkPanel implements \OCP\Dashboard\IPanel {
+
+	/** @var IL10N */
+	private $l10n;
+
+	public function __construct(
+		IL10N $l10n
+	) {
+		$this->l10n = $l10n;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function getId(): string {
+		return 'spreed';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function getTitle(): string {
+		return $this->l10n->t('Talk mentions');
```

Comment:
@jancborchardt this is not what it does atm. It just shows the recent conversations, its not filtering by mentions (which will also be quite empty most of the times (unless the start of the day e.g.)

Reply:
Right, of course. @juliushaertl @nickvergessen is it planned to do the filtering in this PR or in an upcoming one? If in an upcoming one, we can say "Talk conversations" here.

Reply:
like mentioned, I'm not even sure we should filter this way. It will be mostly empty all the time. I think recent conversations is much more helpful then only mentions.

Reply:
Well, _if_ there are mentions, they should for sure be prioritized though.

We have the same issue in the Talk app itself, and in a Dashboard we really need to be concise. :)